### ISSUE TITLE
Spinal epidural abscess 

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6079]
**Preferred term label:**
Spinal epidural abscess 

**Synonyms**
SEA

**Definition (free text, please give PubMed ID)**
A severe pyogenic infection of the epidural space that leads to devastating neurological deficits and may be fatal
PMID: 24007734

**Parent term (use hpo.jax.org/app)**
Abscess

**Comment by pnrobinson** [comment]
adding as HP_0033181
